include/mw/models/crypto/Bech32Address.h

```diff
@@ -19,13 +19,20 @@ class Bech32Address : public Traits::ISerializable, public Traits::IJsonable
 
     static Bech32Address FromString(const std::string& address)
     {
-        auto decoded = bech32::Decode(address);
-        return Bech32Address(decoded.first, decoded.second);
+        auto decoded = bech32::Decode(address); // TODO: Validate checksum
```

Comment:
Do we still need to throw if (decoded.first.empty())?

Reply:
Yea, we should.